doc/tutorials/building_uos_from_clearlinux.rst

```diff
@@ -0,0 +1,160 @@
+.. _build UOS from Clearlinux:
+
+Building UOS from Clear Linux
+#############################
+
+This document builds on the :ref:`getting_started`, 
+and explains how to build UOS from Clear Linux.
+      
+Build UOS image in Clear Linux native
+*************************************
+
+In order to build out the image of UOS, 
+please follow these steps in Clear Linux native:
+
+#. In Clear Linxu native, please install ``ister`` at first, 
+   ``ister`` is a template based installer for linux, 
+   which is inclued in the bundle ``os-installer`` in Clear Linux.
+   For more information about ``ister``, 
+   please visit https://github.com/bryteise/ister.
+
+   .. code-block:: none
+
+      # sudo swupd bundle-add os-installer
+   
+#. After installation is complete, use ``ister.py`` to 
+   generate the image for UOS with the configuration in 
+   ``uos-image.json``:
+
+   .. code-block:: none
+   
+      # cd ~   
+      # sudo ister.py -t uos-image.json
+      
+   An example of the configuration file ``uos-image.json`` 
+   is shown as below:
+
+   .. code-block:: none
+   
+      {
+          "DestinationType" : "virtual",
+          "PartitionLayout" : [ { "disk" : "uos.img",
+                                  "partition" : 1,
+                                  "size" : "512M",
+                                  "type" : "EFI" },
+                                { "disk" : "uos.img",
+                                  "partition" : 2,
+                                  "size" : "32M",
```

Comment:
That seems really low for a swap partition, why is it so small?

Reply:
yes,  looks like swap partition really small, with the UP2 as the example, it has minimal 2G memory, and supposing BIOS, ACRN hypervisor and SOS consumed more than 50%, at lease mem <1G  is for UOS,  why not define swap partition with 1G ?